### PR TITLE
feature/readme.md

### PR DESCRIPTION
**Pull request by alphaseinor**
# Description

Added badges and users to readme, also added some descriptions to the readme file

## Type of change

Feature

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# .codeclimate.yml
+version: "2"
+checks:
+  # Disable checks for similar code.
+  # Because this creates too many false positives.
+  similar-code:
+    enabled: false
+  # Increase the number of lines allowed for a method.
+  # Because this is more reasonable for JavaScript.
+  method-lines:
```
Comment (davidjayfrancis) [inline .codeclimate.yml:10]: Wouldn't this trigger on every component since they are each functions?

File: README.md [hunk 2]
```diff
@@ -1,75 +1,83 @@
...
 
-1️⃣ [Trello Board](🚫add link to trello board here)
 
-1️⃣ [Product Canvas](🚫add link to product canvas here)
 
-1️⃣ [UX Design files](🚫add link to design files here) - 🚫 delete if not applicable
+## Project Overview
 
-🚫 Replace lorem ipsum with a description of your project
+https://trello.com/b/bSVcWjZF/voicer
```
Comment (davidjayfrancis) [inline README.md:39]: I'd add some description to what each of these are and how we use them. "Trello is where we manage our development roadmap and track new features. It's a good place to start if you're looking to contribute." Or something along those lines. Same for the Notion doc link.

File: .codeclimate.yml (hunk 1 above)
Comment (alphaseinor) [reply .codeclimate.yml:10]: this is the boilerplate given to us on the notion document, it's a starting point.

File: README.md (hunk 2 above)
Comment (alphaseinor) [reply README.md:39]: absolutely, I'll see if I can get on that on the next revision

**Pull request merged**